build/pack.ps1

```diff
@@ -89,7 +89,6 @@ Publish-One '../src/QsCompiler/CommandLineTool/CommandLineTool.csproj'
 Publish-One '../src/QuantumSdk/Tools/BuildConfiguration/BuildConfiguration.csproj'
 
 Pack-One '../src/QsCompiler/Compiler/Compiler.csproj' '-IncludeReferencedProjects'
-Pack-One '../src/QsCompiler/CommandLineTool/CommandLineTool.csproj' '-IncludeReferencedProjects'
```

Comment:
I agree with this change, but it may be worth reconsidering in the future whether we want to publish the command-line tool as a tool in its own right.